I cant donate because paypal says something about currency

lifarrx opened the pull request:
![donate](https://user-images.githubusercontent.com/95596140/144868370-71ca6d87-f63f-4181-8369-ba43123a23bf.png)

I don't know if u are brazillian because the name
Mas mano se vocÊ for brasileiro, leia a mensagem em português, quero doar pra ajudar na dev do projeto, me chama no whatsapp 21998266629.

a imagem do paypal que aparece é essa que eu anexei.

Commit:
dinamically load images refactor

Commit:
removesufix method compatibility

Commit:
sendHeroesHome function

Commit:
check if is working and config file options

Commit:
mend

Commit:
mend

Commit:
en readme as default

Commit:
mend

Commit:
green bar image

Commit:
detect missing puzzle piece location

Commit:
detect missing puzzle piece location

Commit:
2 puzzle pieces detection

Commit:
refactor

Commit:
mend

Commit:
aviso

Commit:
aviso

Commit:
aviso

Commit:
aviso

Commit:
aviso

Commit:
first successfull test

Commit:
first successfull test

Commit:
aviso

Commit:
readme

Commit:
Revert "readme"

This reverts commit 051e87b73bb63b73b2846aeeecd9995742229faf.

Commit:
call function from bot main loop

Commit:
call function after clicking back arrow

Commit:
move functions up

Commit:
log

Commit:
solve capcha after connect wallet

Commit:
readme

mpcabete committed:
Check for capcha in main loop and config file

mpcabete committed:
remove prints

mpcabete committed:
error handling

mpcabete committed:
is None

mpcabete committed:
is None

mpcabete committed:
solve capcha antes e depois

mpcabete committed:
Merge branch 'main' of github.com:mpcabete/bombcrypto-bot

mpcabete committed:
readme

Commit:
readme

Commit:
readme

newerton committed:
chore: styling the log

newerton committed:
chore: rollback the check_for_login to 5 minutes

newerton committed:
chore: add folder log

Commit:
merge main

mpcabete committed:
Merge pull request #132 from newerton/feature/styling-log

[Feature] Stylized log

mpcabete committed:
Merge branch 'main' into home-feature

Commit:
merge fixes and disable feature as default

mpcabete committed:
Merge pull request #153 from mpcabete/home-feature

Home feature

Commit:
log fix

Commit:
Merge remote-tracking branch 'origin/home-feature'

Commit:
readme update

Commit:
rm plausible api

Commit:
randomness in intervals and movements

Commit:
nerf randomness

Commit:
randomness cap

Commit:
remove log from randomness call

Commit:
remove log from randomness call

Commit:
requirements

Commit:
radme

Commit:
mend

Commit:
captcha alert

Commit:
captcha alert readme

Commit:
save completed captchas as image

Commit:
save completed captchas as image

Commit:
requirements fix

Commit:
requirements fix

Commit:
requirements fix

Commit:
requirements fix

Commit:
requirements fix

Commit:
requirements fix

Commit:
linux command

Commit:
add folder

Commit:
remove i log

Commit:
Merge branch 'main' into save-captcha

Commit:
readme

Commit:
readme

Commit:
readme

Commit:
readme

Commit:
readme

Commit:
readme

Commit:
readme

Commit:
readme

Commit:
add captcha solver files

Commit:
requirements

Commit:
merge with bot

Commit:
requirements

Commit:
readme

Commit:
readme

Commit:
readme

Commit:
readme

Commit:
readme

Commit:
requirements

Commit:
requirements

newerton commented:
Que PR é essa? Por que você fez tudo isso de commit? Apaga essa branch, o cabete ja finalizo a etapa do novo captcha.

Commit:
remove exit when slider not found

Commit:
remove exit when slider not found

Commit:
slider stuck fix

Commit:
slider stuck fix 2

Commit:
tip

Commit:
milestone

Commit:
milestone x

Commit:
reveal digits

Commit:
print digits

Commit:
print digits

Commit:
print digits

Commit:
print digits

Commit:
first iteration of functional bot

Commit:
readme

Commit:
bkg threshold adjustment

Commit:
pix

Commit:
pix

Commit:
pix

Commit:
pix

Commit:
pix

Commit:
Imagens atualizadas

imagens atualizadas

Nucito committed:
Merge pull request #337 from joaovictor7/feature/new_targets

New images

Commit:
rdm

Commit:
Merge branch 'main' of github.com:mpcabete/bombcrypto-bot

Commit:
digit reveal optimization

Commit:
dinamic threshold in small digits and images replace

filipecavalc committed:
add captcha cache to gitignore

filipecavalc committed:
add some tweaks

filipecavalc committed:
rm pycache

filipecavalc committed:
rm pycache

Commit:
new slider logic

Commit:
new slider logic

Commit:
readme

Nucito committed:
Remove __pycache__

Nucito committed:
Merge pull request #348 from filipecavalc/patch-1

add captcha cache to gitignore

filipecavalc committed:
optmized sign button image

Commit:
slider update

Commit:
slider update

mpcabete committed:
Merge pull request #366 from filipecavalc/main

adicionar imagem de sign otimizada

Commit:
login sequence fix

Commit:
Merge branch 'main' of github.com:mpcabete/bombcrypto-bot

Commit:
readme

Commit:
readme

Nucito committed:
Removing torch from requirements😁

Commit:
corrigido bug que fazia com o click no botão de heróis falhasse

Commit:
imagem do botão 'new-map' atualizada

Nucito committed:
Merge pull request #419 from joaovictor7/bug/click_btn_herois_falhando

Correção da falha dos clicks sobre botão do menu de herois + Atualização de imagem do botão 'new-map'

fguide committed:
Update index.py

Disabling solveCaptcha module
Random wait (1~3 secs) on heroes screen
Offset Fix (issues related)
Green Buttons Clicked counter fix (issues related)

mpcabete committed:
Merge pull request #482 from fguide/main

Remove captcha, offset, random sleep and heroes counter fix

Commit:
remove captcha files

Commit:
remove garbage, and move setup to main function

Commit:
addRandomness docstring

Commit:
load_images docstring

Commit:
show docstring

Commit:
clickBtn docstring and refactor

Commit:
mend

Commit:
mend

Commit:
stale

mpcabete committed:
Merge pull request #494 from mpcabete/refactor

Refactor

Commit:
readme

Commit:
readme

Commit:
cryptip

Commit:
mend

Commit:
cryptip
